src/Plugin/Login.php

```diff
@@ -21,7 +21,7 @@ public function verify( $errors ) {
         $response = $verify->verify_token();
 
         if ( $response === false ) {
-            $errors = new WP_Error('ad_captcha_error', __( '<strong>Error</strong>: Incomplete captcha, Please try again.', 'ad-captcha' ));
+            $errors = new WP_Error('ad_captcha_error', __( '<strong>Error</strong>: Incomplete captcha, Please try again.', 'adcaptcha' ));
```

Comment:
we use `adcaptcha` in the name everywhere else... but `ad_captcha` here and some other places. May be worth being consistent